tasks/backup.yml

```
@@ -0,0 +1,71 @@
+---
+- name: Fail if backup enabled but no output path given
+  ansible.builtin.fail:
+    msg: "database_backup_output_path must be set when database_backup_enabled = true"
+  when:
+    - database_backup_enabled
+    - database_backup_output_path | length == 0
+
+- name: Ensure backup OS user exists
+  ansible.builtin.user:
+    name: "{{ database_backup_owner }}"
+    state: present
+    shell: /usr/sbin/nologin
+    system: true
```

Skip this task if database_backup_owner is root or DB backup isn't enabled.


```suggestion
    system: true
  when:
    - database_backup_enabled
    - database_backup_user != "root"
```

---

Fixed in latest push.